gpflow/probability_distributions.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2017 the GPflow authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.from __future__ import print_function
+
+
+class ProbabilityDistribution:
+    pass
+
+
+class Gaussian(ProbabilityDistribution):
+    def __init__(self, mu, cov):
+        super(Gaussian, self).__init__()
```

Comment:
This line isn't needed.

Reply:
Is it really a bad idea to have it? What if later `ProbabilityDistribution` is given a constructor? Is there a guideline for this?

Reply:
If there's another PR that introduces a constructor to ProbabilityDistribution then that should obviously also add calls to the `super().__init__` to the subclasses... but at this point it's unnecessary, and I'd generally prefer shorter and cleaner code.